micro-benchmarks/DRB172-parallel-orig-no.c

```diff
@@ -0,0 +1,68 @@
+/*
+!!!~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~!!!
+!!! Copyright (c) 2017-20, Lawrence Livermore National Security, LLC
+!!! and DataRaceBench project contributors. See the DataRaceBench/COPYRIGHT file for details.
+!!!
+!!! SPDX-License-Identifier: (BSD-3-Clause)
+!!!~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~!!!
+ */
+/**
+ * BT.C: This file is part of kernel of the NAS Parallel Benchmarks 3.0 bt suit.
+ * Tsan and Intel Inspector can not correctly analysis the nested parallel for, and report a false positive.
+*/
+
+#include <stdio.h>
+
+
+int main(int argh, char* argv[]){
+  int i,j,k,m;
+  int nthreads = 1;
+  double u[12][12][12][5];
+  double tmp1,tmp2,tmp3;
+  double fjac[12][12][12][5][5];
+
+  /*initial m */
+  #pragma omp for
+  for (i = 1; i < 12-1; i++) {
+    for (j = 1; j < 12-1; j++) {
+      for (k = 0; k < 12; k++){
+        for (m = 0; m < 5; m++){
+          u[i][j][k][m] = 2.5;
+        }
+      }
+    }
+  }
+  #pragma omp parallel
+  {
+  #pragma omp for
+  for (i = 1; i < 12-1; i++) {
+    for (j = 1; j < 12-1; j++) {
+      for (k = 0; k < 12; k++) {
+          tmp1 = 1.0 / u[i][j][k][0];
+          tmp2 = tmp1 * tmp1;
+          
+          fjac[i][j][k][0][0] = 0.0;
```

Comment:
This code is still too complex. How many pairs of data races will the tools report? We try our best to mimize the code so a single pair of source locations will be reported to cause data races.

Reply:
Quick check, all this kernel is doing is, in a nested loop of level three, we are having two variables. `temp1 `and `temp2`. It is embarrassingly parallel, the value of `temp1 `is updated and reused to compute `temp2 `without any synchronization construct. Pretty sure this nature we have captured before. Do we really need to include this? Also, variables like `temp3 `are unnecessary. Please remove such vars. 

@chunhualiao can we skip this kernel based on above comment?